src/main/java/net/neoforged/neoforge/event/entity/player/PlayerHeartTypeEvent.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.event.entity.player;
+
+import net.minecraft.client.gui.Gui;
+import net.minecraft.world.entity.player.Player;
+
+/**
+ * Fired by {@link Gui.HeartType#forPlayer} to allow mods to change the heart sprite which is displayed in the player's
+ * health bar.
+ *
+ * <p>
+ * This event is fired only on the client.
```

Comment:
Minor nitpick (not worth changing unless we have other changes to push) - the whitespace line before the `<p>` should be deleted